src/Miso/Effect.hs

```diff
@@ -196,15 +197,13 @@ for actions = withSink $ \sink -> actions >>= flip for_ sink
 withSink :: (Sink action -> JSM ()) -> Effect model action
 withSink f = tell [ f ]
 -----------------------------------------------------------------------------
--- | A synonym for @tell@, specialized to @Effect@
```

Comment:
This comment seems to have been misplaced, as the types of tell and issue don't match. I moved tit to withSink which looks more like a synonym to tell.